python-mach-nix/shell.nix

```diff
@@ -0,0 +1,7 @@
+# legacy shell.nix support flake devShell: https://nixos.wiki/wiki/Flakes
+(import (let lock = builtins.fromJSON (builtins.readFile ./flake.lock);
+in fetchTarball {
+  url =
+    "https://github.com/edolstra/flake-compat/archive/${lock.nodes.flake-compat.locked.rev}.tar.gz";
+  sha256 = lock.nodes.flake-compat.locked.narHash;
```

Comment:
The `sha256` (or any hash argument) to `fetchTarball` is optional, so we don't need to have it here or as a flake input.
This file (and the others) should really just be

```nix
# Flake's devShell for non-flake-enabled nix instances
(import (fetchTarball "https://github.com/edolstra/flake-compat/archive/master.tar.gz") {
  src = ./.;
}).shellNix.default
```
and `flake-compat` inputs should be removed from flake.nix in all the templates.

Reply:
done